### DISCUSSION TITLE
How to get access_token??

### DISCUSSION THREAD

**Discussion by bhagya1101** [directus/directus#16827] [Archive]
How do I get the <token> in access_token ?? 
<img width="1440" alt="Screenshot 2022-12-15 at 2 02 49 PM" src="https://user-images.githubusercontent.com/117433805/207811327-3bd25f69-d9a3-4f3f-9301-8e0084a23f31.png">

**Comment by br41nslug** [comment]
https://docs.directus.io/reference/authentication.html#login

**Reply by Pachat** [reply]
.

**Comment by Pachat** [comment]
Following the doc, can I use a user static token to access the app without login.

1. I set a static token : `ZA0o1ZHBq0y8RnkbrqxfBUi8ikHdckfy` for the user `test@mydomain.com`
2. I try to open the app with `https://mydomain.com/admin/users?access_token=ZA0o1ZHBq0y8RnkbrqxfBUi8ikHdckfy`
but I'm redirected to the login page.

What I want to achieve is a single login between a CMS and the Directus App that are on the same domain (even the same DB).
Could someone put me on the right track?
